lib/game.rb

```diff
@@ -16,4 +18,13 @@ def initialize(game_params)
     @home_goals = game_params[:home_goals].to_i
     @total_goals = @away_goals + @home_goals
   end
+
+  def self.all(games_path)
```

Comment:
Unfortunately the CSV loading will still happen every time you call `Game.all`. Can you think of a way to load the CSV data once, and store it in a way that can be accessed later on?